Makefile.am

```diff
@@ -26,6 +26,17 @@ dist-hook:
 	    echo "OK"; \
 	  fi; \
 	done
+
+# Verify cmake works with the dist tarball.
+cmake_builddir = _build.cmake
+distcheck-hook:
+	$(RM) -rf $(cmake_builddir)
+	mkdir $(cmake_builddir)
```

Comment:
instead of `rm` + `mkdir`, why not just `mkdir -p`?

Reply:
I wanted to make sure the directory was clean, without any cached cmake config from other builds.